pkgs/by-name/br/bristol/package.nix

```diff
@@ -17,31 +19,38 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "mirror://sourceforge/bristol/${pname}-${version}.tar.gz";
-    sha256 = "1fi2m4gmvxdi260821y09lxsimq82yv4k5bbgk3kyc3x1nyhn7vx";
+    hash = "sha256-fR8LvQ19MD/HfGuVSbYXCNeoO03AB4GAEbH1XR+pIro=";
   };
 
   nativeBuildInputs = [
     pkg-config
     copyDesktopItems
   ];
+
   buildInputs = [
     alsa-lib
     libjack2
     libpulseaudio
-    xorg.libX11
-    xorg.libXext
-    xorg.xorgproto
+    libX11
+    libXext
+    xorgproto
   ];
 
-  patchPhase = "sed -i '41,43d' libbristolaudio/audioEngineJack.c"; # disable alsa/iatomic
+  postPatch = ''
+    sed -i '41,43d' libbristolaudio/audioEngineJack.c  # disable alsa/iatomic
```

Comment:
i haven't seen this before, can you link to some references ? eg this looks less obvious than substituteInPlace

Reply:
the `d` instruct sed to delete the selection (here `41,43`). If you have a file with an increasing number on each line and run `sed -i '2,3d' hello`, you would get the following:
```
1
4
5
6
7
```
where line 2 and 4 where removed

Reply:
sorry, I know what it means and does, It seems to be more fragile way to patch than `substiuteInPlace.  What acceptance precedence does it have in nixpkgs ?

Reply:
I agree that sed might be more fragile

the `sed -i '41,43d' libbristolaudio/audioEngineJack.c` was alrady present prior to this pr, so I don't think it should be an big issue (I added `sed -i '35i void doPitchWheel(Baudio *baudio);' bristol/bristolmemorymoog.c` below)

However, adding a patch for every package that have been affected by gcc 14 seems a bit heavy

Reply:
apologies, i didn't see that it was already in the diff, I am pro less changes